ext/oboe_metal/extconf_local.rb

```diff
@@ -0,0 +1,77 @@
+# frozen_string_literal: true
+
+# Copyright (c) 2016 SolarWinds, LLC.
```

Comment:
I see this on all files (with 2016). There is none such for Node and other repos. Not sure it is legally needed these days.

Reply:
it is recommended practice
the year should be the year the code was first written, it does not need to be updated
we have it in the python agent as well